x-pack/elastic-agent/pkg/agent/cmd/enroll_cmd.go

```diff
@@ -865,6 +878,9 @@ func createFleetServerBootstrapConfig(
 	if port == 0 {
 		port = defaultFleetServerPort
 	}
+	if internalPort <= 0 {
+		port = defaultFleetServerInternalPort
```

Comment:
Shouldn't this be `if internalPort > 0 { internalPort = defaultFleetServerInternalPort } `?

Reply:
your condition sets default for a case when somebody tried to configure it

Reply:
🤦‍♂️ Yes. Why is it that on line 878 we have `==` and here `<=`?

But I think the second part still holds, this should be assigned to `internalPort` and not `port`? Maybe I'm blind here too :-D

Reply:
yep, fixed